<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QString与数值直接的转换</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>数值输入和输出</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,2,1,4">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>数量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_number"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>单价</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_price"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1,3">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>总价</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_sumPrice"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>56</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_qDebugTest">
          <property name="text">
           <string>qDebug()测试</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_TotalAmount">
          <property name="text">
           <string>计算总价</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>进制转换</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>十进制</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>二进制</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>十六进制</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit_Dec"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Bin"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Hex"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="rightMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_Dec">
          <property name="text">
           <string>DEC--&gt;其他进制</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Bin">
          <property name="text">
           <string>BIN--&gt;其他进制</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Hex">
          <property name="text">
           <string>HEX--&gt;其他进制</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
